client/src/components/App.js

```diff
@@ -115,15 +117,39 @@ class App extends React.Component {
   };
 
   onClickPrevSong = () => {
-    const { player } = this.props;
+    const { player, decrementCurrentIndex, togglePlaying } = this.props;
 
     if (this.audioRef.current.currentTime >= 3 || player.currentIndex === 0) {
       this.setTimeSong(0);
 
       return;
     }
 
-    this.props.decrementCurrentIndex();
+    decrementCurrentIndex();
+
+    if (!player.isPlaying) {
+      togglePlaying();
+    }
+  };
+
+  onClickPlayTrack = () => {
+    const { setPlayingToTrue, updateSongPlays } = this.props;
+
+    setPlayingToTrue();
+    this.audioRef.current.autoplay = true;
```

Comment:
It might be better to extract this.audioRef.current into a variable say audioRef and use it instead (since it's repeating few times). By doing this you would minimize possibility of making an error, allow easier ref modification in the future and also write less code.